libfreerdp/codec/xcrush.c

```diff
@@ -849,7 +862,7 @@ static int xcrush_decompress_l1(XCRUSH_CONTEXT* xcrush, const BYTE* pSrcData, UI
 		if ((&HistoryPtr[OutputLength] >= HistoryBufferEnd) || (&Literals[OutputLength] > pSrcEnd))
 			return -1012;
 
-		xcrush_copy_bytes(HistoryPtr, Literals, OutputLength);
```

Comment:
Revert this, your assumption that source and destination are different might not hold.

Reply:
No problem too